iroh-gateway/src/main.rs

```diff
@@ -16,20 +22,39 @@ struct Args {
     no_metrics: bool,
 }
 
-#[tokio::main]
-async fn main() {
+#[tokio::main(flavor = "multi_thread")]
+async fn main() -> Result<()> {
+    tracing_subscriber::registry()
```

Comment:
This should no longer be needed.
https://github.com/n0-computer/iroh/blob/main/iroh-metrics/src/lib.rs#L29

Also drops the need to import as a dependency.